page.go

```diff
@@ -217,6 +217,43 @@ type packageData struct {
 	GitTreeName    string
 }
 
+type cacheEntry struct {
+	t  time.Time
+	pd *packageData
+}
+
+var cache map[string]*cacheEntry = make(map[string]*cacheEntry)
+var cacheLock sync.RWMutex
+
+const cacheTTL = 1 * time.Hour
+
+func cacheGet(name string) *packageData {
+	cacheLock.RLock()
+	defer cacheLock.RUnlock()
+	if entry, ok := cache[name]; ok {
+		if time.Since(entry.t) < cacheTTL {
+			return entry.pd
+		}
+		log.Println("ignoring expired cache entry for", name)
+	}
+	return nil
+}
+
+func cacheSet(name string, pd *packageData) {
+	cacheLock.Lock()
+	defer cacheLock.Unlock()
+	if entry, ok := cache[name]; ok {
+		if time.Since(entry.t) < cacheTTL {
+			return
+		}
+		log.Println("updating expired cache entry for", name)
```

Comment:
... one of those.

I suggest dropping both for now.

Reply:
Removed both.